lib/benchpark/experiment.py

```diff
@@ -47,6 +52,19 @@ def get_helper_name_prefix(self):
     def get_spack_variants(self):
         return None
 
+    def set_environment_variable(self, name, value):
+        self.env_vars["set"][name] = value
+
+    def append_environment_variable(self, name, value):
+        self.env_vars["append"][name] = value
+
+    def compute_config_variables(self):
+        pass
+
+    def compute_config_variables_wrapper(self):
+        self.compute_config_variables()
+        return self.variables, self.env_vars
+
```

Comment:
Environment variable implementation from https://github.com/LLNL/benchpark/pull/618/

Reply:
@michaelmckinsey1 please explain what you mean here, and in any other comments/commit messages.  It is impossible to review otherwise.

Reply:
Comments added. From a discussion, the functionality Riyaz wrote in #618 can be used here to append to `PATH` in the jobfiles that Ramble generates. The `append_environment_variable` function generates the expected output in the `ramble.yaml` (https://ramble.readthedocs.io/en/latest/workspace_config.html#environment-variable-control)